src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceServiceTest.kt

```diff
@@ -124,6 +128,12 @@ class LicenceServiceTest {
       additionalConditionUploadDetailRepository,
       auditEventRepository,
       notifyService,
+      omuService,
+      releaseDateService,
+      probationSearchApiClient,
```

Comment:
These four can be removed as only used by licence creation which has been split to own service.

Reply:
Done